pkg/util/resizefs/resizefs_linux.go

```diff
@@ -84,3 +100,19 @@ func (resizefs *ResizeFs) xfsResize(deviceMountPath string) (bool, error) {
 	resizeError := fmt.Errorf("resize of device %s failed: %v. xfs_growfs output: %s", deviceMountPath, err, string(output))
 	return false, resizeError
 }
+
+// findBlockDeviceRescanPath Find the underlaying disk for a linked path such as /dev/disk/by-path/XXXX or /dev/mapper/XXXX
+// will return /sys/devices/pci0000:00/0000:00:15.0/0000:03:00.0/host0/target0:0:1/0:0:1:0/rescan
```

Comment:
The comment is incorrect I think.

Reply:
What is your suggestion? I used this comment as a source: https://github.com/kubernetes/kubernetes/blob/512eccac1f1d72d6d1cb304bc565c50d1f2e295e/pkg/volume/util/device_util_linux.go#L53

Reply:
actually discard this comment. I see that we are resolving the symlink found in `/sys/block/sda/device/rescan` before returning it...